app/assets/scripts/views/countries.js

```diff
@@ -664,20 +691,7 @@ class AdminArea extends SFPComponent {
               <TabPanel>
                 <TabContent title='Overview'>
                   <Fold title='Overview' id='overview'>
-                    <div className='table__basic-grid'>
-                      <BasicTable tableContents={this.renderCountryProfile().countryStatistics} tableTitle='Country Statistics' />
-                      <BasicTable tableContents={this.renderCountryProfile().nationalSociety} tableTitle='National Society' />
-                    </div>
-                    <div className='table__basic-footer'>
-                      <p>
-                        <a href='http://data.ifrc.org/fdrs/' target='_blank'>
-                      Source: {this.renderCountryProfile().source.title}
-                        </a>
-                      </p>
-                      <p className='table__basic-footer-line'>
-                       | Reporting year(s): {Object.keys(this.renderCountryProfile().source.reportingYears).sort().join(', ') || 'N/A'}
```

Comment:
@necoline 
A few suggestions here:
* Should not call `this.renderCountryProfile()` multiple times for the same object but different property (It'll recompute the values every time), instead should have stored the result to a variable and accessed the properties through that variable.
* Should not directly access the property if it can be null (you've returned `null` if the `renderCountryProfile()`) 
* Should check for undefined or provide default value before calling its function (e.g. in `source.reportingYears.sort()`, `reportingYears` could be undefined which will cause an error. Instead we can check for undefined or use default value as `(source.reportingYears || []).sort()`

Reply:
These are very helpful fixes. I really appreciate the review 👍